pkg/backup/item_backupper_test.go

```diff
@@ -242,6 +243,17 @@ func TestBackupItemNoSkips(t *testing.T) {
 				"vol-abc123": {SnapshotID: "snapshot-1", AvailabilityZone: "us-east-1c"},
 			},
 		},
+		{
+			name: "backup fails when takePVSnapshot fails",
```

Comment:
To test your change to item_backupper.go, we need a test case that has this mocked behavior return a non-nil error: `additionalItemBackupper.On("backupItem", mock.AnythingOfType("*logrus.Entry"), test.customActionAdditionalItems[i], item.GroupResource).Return(nil)`.

The test case you've added here is valuable too, so we should keep it, but it doesn't cover your fix: namely, that when we call `ib.additionalItemBackupper.backupItem(log, additionalItem, gvr.GroupResource())`, it checks the returned error and returns it if it's non-nil.

Reply:
Something like this would cover your change: https://gist.github.com/ncdc/12c3b6b6f4db0e5c60f4ecf77e8d85c2

Reply:
Thanks! Updated with this, will ask some questions on Slack too :)